react/ProductSpecification.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { FC, useContext } from 'react'
+import { Specification } from 'vtex.product-context'
```

Comment:
Nice to see this working 😄

Reply:
It doesn't! I created a typing in this project haha